### ISSUE TITLE
can you provide the results of the code?

### ISSUE THREAD

**Issue by zkcys001** [taromakino/abstract_reasoning#1]
How long does it take to train until convergence? and the accuracy of the code? THX

**Comment by taromakino** [comment]
Hi, this is a work in progress; so far I did a sanity check by training until convergence on the simplest 100,000 examples (this included two structure triplets). I was getting ~70% validation error within 10 or so epochs. I tried running the full dataset for a day but the validation error was stuck at I believe ~25%, and I stopped because I am paying for Amazon EC2 out-of-pocket and it was getting expensive. Therefore I'm not 100% yet sure whether the code is accurate or my hyperparameters are set correctly. I need to do a baseline test on the simple 100,000 examples using ResNet or something similar, and see if the RN is out-performing the baseline.

**Comment by zkcys001** [comment]
Thx for your reply! 
I did some experiments about the dataset and find that training on the small kinds of dataset (e.x. shape color progression) can be convergent.
Using full dataset to train may casue the net not convergent. The loss can not degrade.
I am very interested in the work and wang to discuss this work with you in depth.
Can you provide me your email?  Or email me? My email is zkcys001@mail.ustc.edu.cn
